frontend/packages/core/src/WorkflowLayout/index.tsx

```diff
@@ -74,11 +81,11 @@ const WorkflowLayout = ({
   children,
 }: React.PropsWithChildren<LayoutProps>) => {
   const location = useLocation();
-  const workflowPaths = workflow.routes.map(({ path }) => `/${workflow.path}/${path}`);
+  const workflowPaths = workflow?.routes.map(({ path }) => `/${workflow?.path}/${path}`);
```

Comment:
It there is no `workflow.path` what would be rendered?, could it do with a fallback or other handling?

Reply:
I see, because of `workflow` being undefined. 

I think I'm going to change the type signature to reflect that workflow (and the path property) are required.